mindmeld/active_learning/plot_manager.py

```diff
@@ -0,0 +1,474 @@
+import os
+import json
+import logging
+from collections import Counter
+from typing import Dict, List
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .results_manager import create_dir_if_absent
+from ..path import (
+    AL_ACCURACIES_PATH,
+    AL_SELECTED_QUERIES_PATH,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class PlotManager:
+    """Handles plotting. Plots supported:
+    At the domain, intent and entity level:
+        1. Plot Single Epoch (Compares Strategies)
+        2. Plot Average Across Epochs (Compares Strategies)
+        3. Plot All Epochs (Compares Epochs)
+    At the domain and intent level:
+        1. Plot Stacked Bar (Compares Selection Distributions across Iterations)
+    """
+
+    def __init__(self, experiment_dir_path: str):
+        """
+        Args:
+            experiment_dir_path (str): Path to the experiment directory
+        """
+        self.experiment_dir_path = experiment_dir_path
+        self.accuracies_data = self.get_accuracies_json_data()
+        self.queries_data = self.get_queries_json_data()
+
+    # Get JSON Data
+    def get_accuracies_json_data(self) -> Dict:
+        """Loads accuracies.json from the experiment directory path.
+        Returns:
+            data (dict): Data loaded from accuracies.json.
+        """
+        accuracies_json_path = AL_ACCURACIES_PATH.format(
+            experiment_folder=self.experiment_dir_path
+        )
+        with open(accuracies_json_path, "r") as infile:
+            return json.load(infile)
+
+    def get_queries_json_data(self) -> Dict:
+        """Loads queries.json from the experiment directory path.
```

Comment:
what exactly is queries.json used for?

Reply:
Ah, that should be `selected_queries.json`. `selected_queries.json` stores the queries selected by active learning at each iteration. Added a description here